rest-api/test/unit_test/dao_test/participant_summary_dao_test.py

```diff
@@ -218,10 +218,22 @@ def add_sample(participant, test_code, sample_id):
     add_sample(p_mixed_samples, 'NOT1', '44441')
 
     self.dao.update_from_biobank_stored_samples()
+    test_last_modified_doesnt_change_below = p_baseline_samples.lastModified
     self.assertEquals(self.dao.get(p_baseline_samples.participantId).numBaselineSamplesArrived, 2)
     self.assertEquals(self.dao.get(p_mixed_samples.participantId).numBaselineSamplesArrived, 1)
     self.assertEquals(self.dao.get(p_no_samples.participantId).numBaselineSamplesArrived, 0)
 
+    M_baseline_samples = self._insert(Participant(participantId=9, biobankId=99))
+    add_sample(M_baseline_samples, baseline_tests[0], '999')
+    M_first_update = self.dao.get(M_baseline_samples.participantId)
+    self.dao.update_from_biobank_stored_samples()
+    add_sample(M_baseline_samples, baseline_tests[1], '9999')
+    M_second_update = self.dao.get(M_baseline_samples.participantId)
+    self.dao.update_from_biobank_stored_samples()
+
+    self.assertNotEqual(M_first_update.lastModified, M_second_update.lastModified)
+    self.assertEquals(p_baseline_samples.lastModified, test_last_modified_doesnt_change_below)
```

Comment:
You didn't fetch p_baseline_samples again, so this isn't testing the right thing... please add another call to get p_baseline_samples again first.